Src/Fido2/Objects/AuthenticatorData.cs

```diff
@@ -79,7 +79,7 @@ public AuthenticatorData(byte[] rpIdHash, AuthenticatorFlags flags, uint signCou
     /// Backup state is signaled in authenticator data's flags and can change over time.
     /// <see cref="https://w3c.github.io/webauthn/#backup-state"/>
     /// </summary>
-    public bool BackupState => _flags.HasFlag(AuthenticatorFlags.BS);
+    public bool IsBackedUp => _flags.HasFlag(AuthenticatorFlags.BS);
```

Comment:
I was contemplating if we should always keep using Spec terms on places you added `IsBackedUp`.

I decided the readability is worth not aligning to spec (as long as we mention BS/BE in XML).